### PR TITLE
Fix layered defaults for config parameters and linkStatus reporting, add systemd config.

### PR DESCRIPTION
**Pull request by jerearista**
In certain cases, some switch-level config options were not overriding the DEFAULTS section or global configs and linkStatusChanges would not be reported because counter data was reset when an interface status was down.  linkStatusChanges, now has special handling to always be checked, if configured.

A systemd `rphm.service` file was added for customers running on central monitoring workstations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # Remote Port Health Manager (rphm) extension for Arista EOS
 
 ## Overview
-The Remote Port Health Manager (rphm) extension monitors interface counters, then performs actions whenever such counters exceed given thresholds.  The first action generates an SNMP trap.  Other actions could include sending and email or taking corrective action on an erroring port.
+The Remote Port Health Manager (rphm) extension monitors interface counters,
+then performs actions whenever suchelected counters exceed given thresholds.
```
Comment (devrobo) [inline README.md]: suchelected => selected

File: README.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+The default actions are to log a syslog message and generate an SNMP trap.
+Other actions could include sending and email or taking corrective action on an
+erroring port.
 
-This extension may be run directly on Arista EOS devices or on a separate monitoring station and can monitor 1 or many devices based on the config file.
+This extension may be run directly on Arista EOS devices or on a separate
+monitoring station and can monitor 1 or many devices based on the config file.
```
Comment (devrobo) [inline README.md]: 1 or many => 1 or more

File: README.md (hunk 1 above)
Comment (jerearista) [inline README.md]: Corrected in followup commit.

File: README.md (hunk 2 above)
Comment (jerearista) [inline README.md]: Corrected in followup commit.

**Pull request merged**